src/gatsby/src/components/megaMenu.js

```diff
@@ -178,6 +179,11 @@ const MegaMenu = ({ content, menuMode }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [menuElements])
 
+  const isExternalUrl = (url) => {
+    // a URL is considered external if it begins with "http://" or "https://"
+    return /^(http:\/\/|https:\/\/)/.test(url);
```

Comment:
Totally optional, but you could simplify the regular expression to `/^https?:\/\//` (the "http" part with or without an "s")